cmd/kubeapps-apis/buf.gen.yaml

```diff
@@ -48,6 +48,6 @@ plugins:
     opt:
       - outputClientImpl=grpc-web # client implementation it generates
       - esModuleInterop=true # use imports as required in modern ts setups
-      - useOptionals=true # use optional TypeScript properties instead of undefined
+      - useOptionals=messages # use optional TypeScript properties instead of undefined
```

Comment:
I got a deprecation message here saying `true` will be no longer accepted in upcoming versions. The drop-in replacement is `messages` as they mentioned in their docs.